.github/workflows/mongodb.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow tests the feature of using an external MongoDB for tests
+name: Java Test MongoDB
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  # Test with real MongoDB
+  test-mongodb:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    env:
+      TEST_MONGODB_CONNECTION_STRING: mongodb://test:example@localhost:27017/testdb?authSource=admin
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@5ffc13f4174014e2d4d4572b3d74c3fa61aeb2c2 # v3.11.0
+        with:
+          distribution: 'temurin'
+          java-version: 17
+          cache: 'gradle'
+
+      - name: Start MongoDB
+        run: docker run -d -e MONGO_INITDB_ROOT_USERNAME=test -e MONGO_INITDB_ROOT_PASSWORD=example -p 27017:27017 --name test_mongo mongo:4.4
```

Comment:
We could go to MongoDB 5 like in dw-commons.

Reply:
Funny … I had that before and switched back because I found 4.4 in the task. Thought it has a reason.

Reply:
I would just use the same setup like in DW. So let's update to 5 since this is [now](https://github.com/SDA-SE/sda-dropwizard-commons/commit/d1c7b6d8abf258f58659b1bc73c6385c4ce7cba6) the version that we use there.

Reply:
bumped to 5.0